### PR TITLE
Fix the system-tag cli help default values after #1148

### PR DESCRIPTION
**Pull request by MStoykov**

### CODE REVIEW

**Comment by MStoykov** [comment]
There is already a `String() string` method from the `enumer` so I decided to leave that one alone and add another implementation just for this case.
The naming was problematic, I first called it `HelpString`, but later decided that it's stupid.

File: stats/system_tag.go [hunk 1]
```diff
@@ -70,6 +71,17 @@ func (i *SystemTagSet) Map() TagSet {
 	return m
 }
 
+// SetString is returns comma separeted list of the string representation of all values in the set
```
Comment (cuonglm) [inline stats/system_tag.go]: Typo in "is returns".

And also, should this method named `List()`, for alignment with `Map()` above/

Comment (MStoykov) [reply stats/system_tag.go]: Fixed the typo and did some .. performance optimizations (not actually needed I am sure). 

I don't think it's worth it to add a generic way to get a list of the set values from the SystemTags ... this is literally the only place where it's used ...

File: stats/system_tag.go [hunk 2]
```diff
@@ -70,6 +70,17 @@ func (i *SystemTagSet) Map() TagSet {
 	return m
 }
 
+// SetString returns comma separeted list of the string representation of all values in the set
```
Comment (imiric) [inline stats/system_tag.go]: Typo: "separeted"

**Pull request merged**